### PR TITLE
Fix Sentry DSN

### LINKED ISSUE #1655: SENTRY_DSN can not be defined
When trying to set SENTRY_DSN I get the following error
```
docker compose logs jicofo       
jitsi-jicofo-1  | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
jitsi-jicofo-1  | [s6-init] ensuring user provided files have correct perms...exited 0.
jitsi-jicofo-1  | [fix-attrs.d] applying ownership & permissions fixes...
jitsi-jicofo-1  | [fix-attrs.d] done.
jitsi-jicofo-1  | [cont-init.d] executing container initialization scripts...
jitsi-jicofo-1  | [cont-init.d] 01-set-timezone: executing... 
jitsi-jicofo-1  | [cont-init.d] 01-set-timezone: exited 0.
jitsi-jicofo-1  | [cont-init.d] 10-config: executing... 
jitsi-jicofo-1  | panic: render template error: template: /defaults/logging.properties:1:38: executing "/defaults/logging.properties" at <toBool>: error calling toBool: strconv.ParseBool: parsing "https://<redacted>": invalid syntax
jitsi-jicofo-1  | 
jitsi-jicofo-1  | goroutine 1 [running]:
jitsi-jicofo-1  | main.templateExecute(0xc0000b2580?, {0x7ffc4ab18e22, 0x1c})
jitsi-jicofo-1  |       /Users/runner/work/tpl/tpl/main.go:48 +0x1b5
jitsi-jicofo-1  | main.main()
jitsi-jicofo-1  |       /Users/runner/work/tpl/tpl/main.go:69 +0x226
jitsi-jicofo-1  | [cont-init.d] 10-config: exited 0.
jitsi-jicofo-1  | [cont-init.d] done.
jitsi-jicofo-1  | [services.d] starting services
jitsi-jicofo-1  | [services.d] done.
```

`logging.properties` is then empty.

jicofo here just stands as example. The same code is present in jigasi and jvb

### PR DESCRIPTION
**Pull request by DanielMcAssey**
Actually fix #1655 

This supports having individual DSNs

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -353,7 +353,7 @@ services:
             - MAX_BRIDGE_PARTICIPANTS
             - OCTO_BRIDGE_SELECTION_STRATEGY
             - PROSODY_VISITORS_MUC_PREFIX
-            - SENTRY_DSN="${JICOFO_SENTRY_DSN:-0}"
```
Comment (krombel) [inline docker-compose.yml:356]: This removes support for setting individual DSN values.

SENTRY_DSN is the name the environment variable the sentry library enforces to use.

File: jicofo/rootfs/defaults/logging.properties [hunk 2]
```diff
@@ -1,7 +1,9 @@
-{{ if .Env.SENTRY_DSN | default "0" | toBool }}
+{{ $SENTRY_DSN := .Env.SENTRY_DSN | default .Env.JICOFO_SENTRY_DSN -}}
```
Comment (krombel) [inline jicofo/rootfs/defaults/logging.properties:1]: This does not help. JICOFO_SENTRY_DSN is not evaluated (as only SENTRY_DSN is)

**Review by krombel — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (DanielMcAssey) [reply docker-compose.yml:356]: The variable is read during templating, please see the logging.properties

File: jicofo/rootfs/defaults/logging.properties (hunk 2 above)
Comment (krombel) [reply jicofo/rootfs/defaults/logging.properties:1]: I would suggest to just strip the ` | default "0" | toBool`-part.

This is at least working in my environment

**Comment by krombel** [comment]
As you just changed this PR to beeing a draft I PRd my changes I implemented and tested in my env.

I hope you are fine with it. For reference: #1659

**Pull request closed without merging**